common/main.c

```diff
@@ -5,19 +5,32 @@
 
 #include <compiler.h>
 #include <console.h>
+#include <debug.h>
 #include <dm.h>
 #include <stdbool.h>
 #include <work.h>
+#include <drivers/watchdog.h>
 #include <platform/devices.h>
 
+#define WDOG_TIMEOUT (5 * 1000 * 1000) * 24 /* 5 seconds */
+
 noreturn void main(void);
 
 noreturn void
 main(void)
 {
+	struct device *watchdog;
+
 	console_init(DEV_UART0);
 	dm_init();
 
+	/* Enable watchdog. */
+	if ((watchdog = dm_get_by_class(DM_CLASS_WATCHDOG))) {
+		watchdog_enable(watchdog, WDOG_TIMEOUT);
+		info("Trusted watchdog enabled.");
```

Comment:
The tiniest of nits, but none of the other messages have periods. This one should match.

Reply:
Done.